### ISSUE TITLE
Dropbox Paper icon

### ISSUE THREAD

**Issue by dinosw** [morkro/papyrus#12]
How come you do not use the Dropbox Paper Icon for the exe?
It would make it easier to identify the app, instead of now where it uses the Dropbox icon.
If you do not have the icon, then I have it here.: [Dropbox Paper.ico](https://www.mediafire.com/?j5y5y1nxly7xjjn).
I have simply converted the image used on the Google Play Store.

**Comment by morkro** [comment]
Hey 👋 
Which icon do you see for the exe? Isn't [this one](https://raw.githubusercontent.com/morkro/papyrus/master/static/dropbox-windows.ico)?

**Comment by dinosw** [comment]
Yes, I see the dropbox icon, but it would be easier if the exe had the Dropbox Paper icon instead.: [Dropbox Paper Icon](https://lh3.googleusercontent.com/w3MA6GeOjbP2tanUnp6J6I1t-wMFzeAuuN7rvEMXCYDKntCjTiDtgdieRWNq8RELhyQQ=w300-rw)

**Comment by dinosw** [comment]
Screenshot showing the dropbox favicon in a browser tab.: [Screenshot](https://goo.gl/VpuZvI)
